kmcd-driver.md

```diff
@@ -155,14 +155,17 @@ Base Data
 **TODO**: Should we add operators like `+Wad` which emulate the precision limits described in `makerdao/dss/DEVELOPING.md`, or assume the abstract model to be inifinite precision?
 
 ```k
-    syntax Wad ::= Int
- // ------------------
+    syntax Wad
+    syntax Wad = Rat
+ // ----------------
 
-    syntax Ray ::= Int
- // ------------------
+    syntax Ray
+    syntax Ray = Rat
+ // ----------------
 
-    syntax Rad ::= Int
- // ------------------
+    syntax Rad
+    syntax Rad = Rat
+ // ----------------
```

Comment:
Why is this happenning? We don't need to declare `syntax Rad` ahead of time I don't think?

Reply:
there was a bug in the sort synonym implementation. seems it is fixed now https://github.com/kframework/k/commit/da5cd7077914dd5c02b7431dc43f2cdfb6b7a918